packages/webapp/src/components/Modals/WaterUsageCalculatorModal/index.jsx

```diff
@@ -185,6 +200,7 @@ const WaterUseDepthCalculator = ({ system, setTotalWaterUsage, totalWaterUsage,
         max={999999.99}
         system={system}
         control={control}
+        hideLineUnitSeparator={true}
```

Comment:
@wasswarichard 

Based off the discussion today please remove this line all references to this 'hideLineUnitSeperator'